src/EditorFeatures/Core.Wpf/InlineHints/InlineHintsTag.cs

```diff
@@ -65,6 +65,8 @@ internal class InlineHintsTag : IntraTextAdornmentTag
             // information in the Border_ToolTipOpening event handler
             adornment.ToolTip = "Quick info";
             adornment.ToolTipOpening += Border_ToolTipOpening;
+
+            adornment.MouseLeftButtonDown += Adornment_MouseLeftButtonDown;
```

Comment:
do we need soemthing that disconnects this event handler?  how do we avoid leaks?

Reply:
I'm not sure, we do not disconnect the tooltipopening event either. What scenarios would require disconnects?

Reply:
This should be fine, I think. The lifetime of the tag and adornment are already tied together, right? If this was subscribing to a TextView event or something, then absolutely we'd need an unsubscribe somewhere.

Reply:
Yeah they're tied together, they both get removed and recreated at the tags changed event, classification format map changed event, etc.

Reply:
soudns good.